content/how-to-analyse-haskell-performance/index.md

```diff
@@ -0,0 +1,22 @@
++++
+title = "A Beginner's Guide To Analysing Haskell Application Performance"
+date = 2024-11-27
+[taxonomies]
+authors = ["Hécate"]
+categories = ["Cabal"]
+tags = ["Performance", "Profiling"]
++++
+
+The Cabal manual now has a guide on how to analyse the performance of Haskell applications.
+
+<!-- more -->
+
+This guide was contributed very generously by Malte Neuss. It describes the various options you can set in your `cabal.project` file in order to get a time or space profile of your application.
```

Comment:
Maybe instead of mentioning `cabal.project` (since we don't use that in the guide), just write
`...options you can set in your Cabal project in order to...`.

The rest looks good.

Reply:
Unless I'm severely mistaken, the guide does mention it:

> This command creates a cabal.project.local file with the following content:
> profiling: True

Reply:
I had thought that we did mention `cabal.project.local` only and not literally `cabal.project, but we do mention the difference in a note.

Reply:
Feel free to resolve. I can't.